src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -15,8 +15,9 @@
 
 
 def get_link(
-    config: RepositoryProjectPathConfig, filepath: str, default: str, version: Optional[str] = None
+    config: RepositoryProjectPathConfig, filepath: str, version: Optional[str] = None
```

Comment:
This is just making the signature of this function take one less argument since it is already part of the `config` variable.